src/domain/hanke/edit/components/Haittojenhallintasuunnitelma.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Box } from '@chakra-ui/react';
+import { FORMFIELD, HankeDataFormState } from '../types';
+import { HAITTOJENHALLINTATYYPPI, HankeData, HankeIndexData } from '../../../types/hanke';
+import TextArea from '../../../../common/components/textArea/TextArea';
+import { sortedLiikenneHaittojenhallintatyyppi } from '../utils';
+import useFieldArrayWithStateUpdate from '../../../../common/hooks/useFieldArrayWithStateUpdate';
+import HankealueMap from '../../../map/components/HankkeenHaittojenhallintasuunnitelma/HankealueMap';
+import VectorSource from 'ol/source/Vector';
+import useAddressCoordinate from '../../../map/hooks/useAddressCoordinate';
+
+type Props = {
+  hanke: HankeData;
+  index: number;
+};
+
+const Haittojenhallintasuunnitelma: React.FC<Props> = ({ hanke, index }) => {
+  const { t } = useTranslation();
+  const { fields: hankealueet } = useFieldArrayWithStateUpdate<HankeDataFormState, 'alueet'>({
+    name: FORMFIELD.HANKEALUEET,
+  });
+  const hankealue = hankealueet[index];
+  const haittojenhallintatyypit = sortedLiikenneHaittojenhallintatyyppi(
+    hankealue.tormaystarkasteluTulos as HankeIndexData,
+  );
+  const [drawSource] = useState<VectorSource>(new VectorSource());
+  const addressCoordinate = useAddressCoordinate(hanke.tyomaaKatuosoite);
+
+  return (
+    <div>
+      <Box mt="var(--spacing-m)" mb="var(--spacing-xs)" fontWeight="bold">
```

Comment:
Would be better for semantics if this was heading element instead of div, maybe `<Box as="h4">`

Reply:
Changed it.